JS/script.js

```diff
@@ -1,61 +1,104 @@
 
-    let canvas = canvasRow.repeat(row)
+    let canvas =  canvasRow.repeat(row)
 
     document.querySelector('.canvas').innerHTML = canvas
 
 }
 
-let painting
-let clickdown
 
-/* 
-* The two global events below are designed to ensure that users can 
+function setCanvasColor(color) {
+    document.querySelectorAll('.row .pixel').forEach(function (pixel) {
+        pixel.style.backgroundColor = color
+    })
+    document.querySelector('.canvas').style.backgroundColor = color
+}
+
+/*
+* The two global events below are designed to ensure that users can
 * click elsewhere on the page, and when they drag onto the canvas, the
-* paint will be flowing. 
+* paint will be flowing.
 * Conversely, if the user unclicks when not on the canvas, the user will
-* not be painting when they return the mouse to the canvas unless they 
-* click again. 
+* not be painting when they return the mouse to the canvas unless they
+* click again.
 */
 
-document.querySelector('html').addEventListener('mousedown', function () {
+let clickdown, painting
+
+document.querySelector('html').addEventListener('mousedown', function() {
     clickdown = true
 })
 
-document.querySelector('html').addEventListener('mouseup', function () {
+document.querySelector('html').addEventListener('mouseup', function() {
     clickdown = false
 })
 
-document.querySelector('.canvas').addEventListener('mouseleave', function () {
+document.querySelector('.canvas').addEventListener('mouseleave', function() {
     painting = false
 })
 
-document.querySelector('.canvas').addEventListener('mouseenter', function () {
+document.querySelector('.canvas').addEventListener('mouseenter', function() {
     if (clickdown) {
         painting = true
     }
 })
 
-document.querySelectorAll('.pixel').forEach(function(pixel) {
-    pixel.addEventListener('mousedown', function() {
-        painting = true
-        this.style.backgroundColor = "#000000	"
-    })
-    pixel.addEventListener('mousemove', function() {
-        if (painting === true && clickdown === true) {
-            this.style.backgroundColor = "#000000	"
-        }
-    })
-    pixel.addEventListener('mouseup', function() {
-        painting = false
-    })
+document.querySelectorAll('.row .pixel').forEach(function(pixel) {
+    setTimeout(function() {
+        pixel.addEventListener('mousedown', function() {
+            painting = true
+            this.style.backgroundColor = paintColor
+        })
+        pixel.addEventListener('mousemove', function() {
+            if (painting === true && clickdown === true) {
+                this.style.backgroundColor = paintColor
+            }
+        })
+        pixel.addEventListener('mouseup', function() {
+            painting = false
+        })
+    }, 0)
+})
+
+let drawer = document.querySelector('#drawer')
+let eraser = document.querySelector('#eraser')
+
+drawer.addEventListener('click', function() {
+    drawer.style.backgroundColor = '#a9a9a9'
```

Comment:
might be beneficial to make these colours variables so you dont have to change them in 2 places